docs/CATALOG_ITEMS.md

```
@@ -0,0 +1,305 @@
+# Catalog Items
+
+## Overview
+
+OSAC uses a three-level hierarchy to provision infrastructure resources:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)